example/src/screens/calendars.js

```diff
@@ -355,16 +355,12 @@ const CalendarsScreen = () => {
         <Calendar
           style={[
             styles.calendar,
-            {
-              height: 250,
-              borderBottomWidth: 1,
-              borderBottomColor: 'lightgrey'
-            }
+            styles.customCalendar
           ]}
           dayComponent={({date, state}) => {
             return (
               <View>
-                <Text style={{textAlign: 'center', color: state === 'disabled' ? 'gray' : 'purple'}}>{date.day}</Text>
+                <Text style={[styles.customDay, {color: state === 'disabled' ? 'gray' : 'purple'}]}>{date.day}</Text>
```

Comment:
you can improve here a little and create `styles.disableDay` & `styles.day` and pass them in the ternary.

Reply:
Done